spec/services/invitations/generate_letter_spec.rb

```diff
@@ -0,0 +1,50 @@
+describe Invitations::GenerateLetter, type: :service do
+  subject do
+    described_class.call(
+      invitation: invitation
+    )
+  end
+
+  let!(:applicant) { create(:applicant) }
+  let!(:invitation) { create(:invitation, content: nil, applicant: applicant, format: "postal") }
+
+  describe "#call" do
+    it("is a success") { is_a_success }
+
+    it "generates the pdf string" do
+      subject
+      expect(invitation.content).not_to eq(nil)
```

Comment:
nitpick: on pourrait rajouter un matcher sur le content `expect(invitation.content) to match(/.../)`